### PR TITLE
Create guide_invoiceplane.rst

### PR DESCRIPTION
**Pull request by thomas-kammerer**

### CODE REVIEW

File: source/guide_invoiceplane.rst [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: business
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/invoiceplane.png
```
Comment (luto) [inline source/guide_invoiceplane.rst]: This file is missing from the PR.

File: source/guide_invoiceplane.rst [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+ [isabell@stardust html]$ unzip v1.5.10
+ [isabell@stardust html]$ rm v1.5.10
+ [isabell@stardust html]$ 
+
+Configuration
+=============
+
+Create the database, copy the configuration template, rename the ``htaccess`` file and add your URL ``https://isabell.uber.space`` in the ``ipconfig.php`` file under ``IP_URL``.
+
+ [isabell@stardust html]$ mysql -e "CREATE DATABASE ${USER}_ip"
```
Comment (luto) [inline source/guide_invoiceplane.rst]:
```suggestion

.. code-block:: console

 [isabell@stardust html]$ mysql -e "CREATE DATABASE ${USER}_ip"
```

The formatting for this is broken.

File: source/guide_invoiceplane.rst [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ cp ipconfig.php ~
+ [isabell@stardust html]$ rm -rf * .*
+
+Proceed with the installation steps from here and move back your config file.
+
+ [isabell@stardust html]$ mv ~/ipconfig.php ./
```
Comment (luto) [inline source/guide_invoiceplane.rst]:
```suggestion

.. code-block:: console

 [isabell@stardust html]$ mv ~/ipconfig.php ./
```

The formatting for this is a bit broken.

File: source/guide_invoiceplane.rst [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+=============
+
+Create the database, copy the configuration template, rename the ``htaccess`` file and add your URL ``https://isabell.uber.space`` in the ``ipconfig.php`` file under ``IP_URL``.
+
+ [isabell@stardust html]$ mysql -e "CREATE DATABASE ${USER}_ip"
+ [isabell@stardust html]$ cp ipconfig.php.example ipconfig.php
+ [isabell@stardust html]$ mv htaccess .htaccess
+ [isabell@stardust html]$
+
+After the installation you need to open https://isabell.uber.space/index.php/setup in your browser to finish your setup.
```
Comment (luto) [inline source/guide_invoiceplane.rst]: Opening the site always redirects to `http://127.0.0.1/...` for me, unless I set `IP_URL=https://isabell.uber.space`.

Comment (thomas-kammerer) [reply source/guide_invoiceplane.rst]: That part is described in the configuration section.

Comment (luto) [reply source/guide_invoiceplane.rst]: ooh, up there. sorry!

**Pull request merged**